datalad/install.py

```diff
@@ -318,12 +312,37 @@ def _install_via_conda(self, version, tmpdir):
             check=True,
         )
         if self.annex_bin != conda_bin:
-            os.makedirs(self.annex_bin, exist_ok=True)
-            with dirchanged(self.annex_bin):
-                for fname in glob("../miniconda/bin/git-annex*"):
-                    os.symlink(fname, os.path.basename(fname))
+            annex_bin = Path(self.annex_bin)
+            annex_bin.mkdir(parents=True, exist_ok=True)
+            for p in Path(conda_bin).glob("git-annex*"):
+                (annex_bin / p.name).symlink_to(p.resolve())
         self.post_install()
 
+    def install_miniconda(self, miniconda_path, batch=False):
+        miniconda_script = "Miniconda3-latest-Linux-x86_64.sh"
+        log.info("Downloading and running miniconda installer")
+        with tempfile.TemporaryDirectory() as tmpdir:
+            script_path = os.path.join(tmpdir, miniconda_script)
+            subprocess.run(
+                [
+                    "wget",
```

Comment:
looking forward to glorious windows times etc, might be worth to redo this avoiding `wget` with pure python urllib.  Some linux boxes also might not have `wget`.

Reply:
Done.